seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java

```diff
@@ -124,10 +125,26 @@ public void write(SeaTunnelRow element) throws IOException {
             return;
         }
         dorisStreamLoad.writeRecord(serialize);
+        if (!dorisConfig.getEnable2PC()
+                && dorisStreamLoad.getRecordCount() >= dorisConfig.getBatchSize()) {
+            flush();
+            startLoad(labelGenerator.generateLabel(lastCheckpointId));
```

Comment:
Is it okay to flush multiple times between checkpoints? Will the flush label be the same for multiple times？

Reply:
It will only worked when disabled 2pc, therefore, it has no impact on the consistency of the data. The flush label will use current time when on disabled 2pc. Please refer https://github.com/apache/seatunnel/blob/c324d663b4661e315361b2a498cab278458021f5/seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/LabelGenerator.java#L33

PS: The reason for add this PR is that we had disabled checkpoint in batch mode, so too much data may be written at the end of the task. Has no impact on our current consistency semantics.

Reply:
ok,I think doris.batch.size needs careful explanation, such as the timing of his role?

Reply:
how about now?

Reply:
I think it's good